### ISSUE TITLE
[Feature Request]: Git Plugin: make git commit info select-able and copy-able 

### ISSUE THREAD

**Issue by UffeJakobsen** [eranif/codelite#3532]
### Describe your idea in details

[Feature Request]: Git Plugin: make git commit info select-able and copy-able

Right now the git commit info can only be visually inspected (seen) - it cannot be copied - 
It would be usable if the commit SHA and/or other things could be copied into the copy-paste buffer - so the info could be used in other programs

**Comment by eranif** [comment]
I am not sure I understand which "commit info" you are referring to. 

This?
<img width="711" alt="Capture" src="https://github.com/user-attachments/assets/90bbba84-d9d1-47d1-8432-465c537c73ce">

**Comment by UffeJakobsen** [comment]
Yes - exactly that - an example of usage is for situations where you'll want to copy the SHA for further inspection in another GUI
